src/django/api/serializers/user/user_profile_serializer.py

```diff
@@ -72,6 +72,7 @@ def get_facility_lists(self, user):
                     source__contributor=contributor,
                     source__is_active=True,
                     source__is_public=True,
+                    status=FacilityList.APPROVED,
```

Comment:
@roman-stolar , Frankly saying I doubt that only approved need to be shown. It's not clear from the task. Let's clarify with @mariel-oar